### PR TITLE
feat: learn page "max" breakpoint layout

### LINKED ISSUE #1761: Display new max size breakpoint on Learn page
Adapt the "learn" page to support the new layout for the "max" breakpoint.

---

Related to #1747

### PR DESCRIPTION
**Pull request by eddybrando**
This PR adapts the learn page to support the new layout for the "max" breakpoint.

---

Closes #1761

### CODE REVIEW

File: components/learn/TheLearningResourceList.vue [hunk 1]
```diff
@@ -1,9 +1,9 @@
 <template>
   <section class="the-learning-resources-list">
-    <div class="the-learning-resources-list__container">
+    <div class="bx--grid">
       <h2>Learning resources</h2>
       <div class="bx--row the-learning-resources-list__introduction">
```
Comment (y4izus) [inline components/learn/TheLearningResourceList.vue:5]: You can delete the `the-learning-resources-list__introduction` class

File: components/ui/AppCard.vue [hunk 2]
```diff
@@ -94,11 +95,18 @@ export default class AppCard extends Vue {
...
 
   &__content {
-    padding: $spacing-05 $spacing-05 $spacing-05 $spacing-07;
+    padding: $spacing-05 $spacing-06;
     flex: 1;
     display: flex;
     flex-direction: column;
-    justify-content: space-between;
+
+    @include mq($from: medium) {
```
Comment (y4izus) [inline components/ui/AppCard.vue:103]: These two `mq` are not related with this PR, could you delete them?

File: components/ui/AppFiltersResultsLayout.vue [hunk 3]
```diff
@@ -29,21 +29,38 @@ export default class AppFiltersResultsLayouts extends Vue {
...
 
-  &__results-item {
+  ::v-deep &__results-item {
     margin-bottom: $layout-02;
 
     @include mq($until: large) {
       margin-bottom: $layout-01;
     }
+
+    @include mq($from: medium) {
```
Comment (y4izus) [inline components/ui/AppFiltersResultsLayout.vue:47]: At least at the moment, the card's height should be adapted to the content, so please, delete these `min-height`

**Review by y4izus — CHANGES_REQUESTED** [review]
Thank you! please, take a look at my comments and ask me for another review :)

File: pages/learn/index.vue [hunk 4]
```diff
@@ -24,27 +24,35 @@
...
+          v-for="resource in filteredLearningResources"
+          :key="resource.path"
+          class="bx--col-sm-4 bx--col-xlg-8"
+        >
+          <AppCard
+            :image="resource.image"
+            :title="resource.title"
+            :to="resource.to"
+            :cta-label="resource.ctaLabel"
+            class="app-filters-results-layout__results-item app-filters-results-layout__results-item_2-cols-xlg"
```
Comment (y4izus) [inline pages/learn/index.vue:50]: Why did you add this `app-filters-results-layout__results-item_2-cols-xlg`? It's not enough with adding

```
@include mq($from: x-large) {
     height: calc(100% - #{$layout-02});
 }
```

to the `&__results-item`?

File: pages/learn/index.vue [hunk 5]
```diff
@@ -153,3 +160,15 @@ export default class LearnPage extends QiskitPage {
   }
 }
 </script>
+
+<style lang="scss" scoped>
+.app-filters-results-layout {
```
Comment (y4izus) [inline pages/learn/index.vue:165]: Why did you add this code here instead of in the `AppFiltersResultsLayout` component?

**Review by y4izus — COMMENTED** [review]
Thank you! I have two more comments and we finish :)

File: pages/learn/index.vue (hunk 4 above)
Comment (eddybrando) [reply pages/learn/index.vue:50]: Because `app-filters-results-layout__results-item` is also used in the **Events Page**, which currently doesn't have the "max" layout, so it would look like this:

Screenshot 2021-05-25 at 12 57 11

File: pages/learn/index.vue (hunk 5 above)
Comment (eddybrando) [reply pages/learn/index.vue:165]: Good catch!

It makes sense for that definition live in this file, but the name of the class should actually be `learn-page__careful-explanation`.

I'll change it. Thank you! 🙂

File: pages/learn/index.vue (hunk 4 above)
Comment (y4izus) [reply pages/learn/index.vue:50]: great! thank you!

**Review by y4izus — APPROVED** [review]
Great Eddy!! thank you very much!!
